src/ServiceStack.OrmLite/Support/LoadList.cs

```diff
@@ -55,7 +55,7 @@ protected string GetRefListSql(ModelDefinition refModelDef, FieldDefinition refF
                          $"IN ({subSql})";
 
             if (OrmLiteConfig.SqlSelectRefFilter != null)
-                sqlRef = OrmLiteConfig.SqlSelectRefFilter?.Invoke(refModelDef.ModelType, sqlRef);
+                sqlRef = OrmLiteConfig.SqlSelectRefFilter.Invoke(refModelDef.ModelType, sqlRef);
 
```

Comment:
`.Invoke()` should be removed as well, e.g:

```csharp
 sqlRef = OrmLiteConfig.SqlSelectRefFilter(refModelDef.ModelType, sqlRef);
```

Reply:
Yep, sorry about that was carry over from other code tests. fixed